### PR TITLE
Added computer ordering info

### PR DESCRIPTION
**Pull request by rando2**
Someone who has built a desktop might want to add onto this

### CODE REVIEW

File: extras/infrastructure.md [hunk 1]
```diff
@@ -38,6 +38,12 @@ If you do so, please also send the details of your order (such as Amazon URLs) t
 
 For the delivery address, use Casey's office (10-131 Smilow Center for Translational Research, 3400 Civic Center Boulevard, Philadelphia, PA 19104).
 
+## Purchasing Computers
+
+If you would like to purchase parts to assemble a desktop, follow the Purchasing procedures above.
+
+Other computers and technological equipment are ordered through the [Computer Connection](https://cms.business-services.upenn.edu/computerstore/). Speak with Paul Culkin (`paul2@upenn.edu`) to generate a quote. The quote Paul provides should be forwarded to Casey, Maria Guglietti (`gugliett@upenn.edu`) and Roz (`rucker@upenn.edu`). You will likely need to fill out a [pharmacology requisition form](https://github.com/greenelab/onboarding/blob/master/forms-and-docs/regular-vendor-purchase-form.xlsx) in consultation with Roz.
```
Comment (cgreene) [inline extras/infrastructure.md]: Can you make this one sentence per line for easier diffs?

**Review by cgreene — COMMENTED** [review]
This looks good to me, but can you make it one sentence per line so it's easier to look at future changes? Thanks!

**Comment by rando2** [comment]
I think I fixed it now, but I am not familiar with how to edit a pull request, so please just let me know if it didn't work!

**Review by cgreene — APPROVED** [review]

**Comment by cgreene** [comment]
Thanks!

**Pull request merged**